qupath-gui-fx/src/main/java/qupath/lib/gui/scripting/languages/ScriptLanguageProvider.java

```diff
@@ -106,8 +106,8 @@ private static List<ScriptLanguage> getDefaultLanguages() {
 	}
 	
 	
-	private static ExtensionClassLoader getExtensionClassLoader() {
-		return ExtensionClassLoader.getInstance();
+	private static ClassLoader getExtensionClassLoader() {
+		return QuPathGUI.getInstance().getExtensionCatalogManager().getClassLoader();
```

Comment:
@Rylern I missed this when merging the PR: this breaks command line scripts,  because then `QuPathGUI` doesn't exist.

We need a method of getting the extension `ClassLoader` that has no dependency on the UI.